### DISCUSSION TITLE
vc-datasource-czml组件读取czml文件调用createCesiumObject函数报错Uncaught (in promise) ReferenceError: Cesium is not defined

### DISCUSSION THREAD

**Discussion by xrksama** [zouyaoji/vue-cesium#566] [Q&A]
![image](https://github.com/zouyaoji/vue-cesium/assets/27134996/d8f6cafa-4b75-4701-aa1c-fbcd7994afd0)
我想像 Cesium.CzmlDataSource.load函数那样读取czml文件后能返回一个promise对象，于是在vc-datasource-czml组件读取后调用了其中的createCesiumObject函数，调用方式为
![image](https://github.com/zouyaoji/vue-cesium/assets/27134996/61fd46db-7e69-4868-88e6-671b6965c24d)

**Comment by zouyaoji** [comment]
（1）vc-datasource-czml绑定ready事件拿到数据源实例。
（2）vc-datasource-czml绑定ref通过ref的getCesiumObject方法拿到数据源实例。

任选其一
